compileui.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+"""This script runs a command that compiles the .ui file to a .py file.
+
+Must be run from .../HeadCircumferenceTool
+
+You must have PySide6 and the pyside6-uic command installed"""
+
+import os
+from src.utils.constants import UI_FILE_PATH, COMPILED_UI_FILE_PATH
+
+os.system(f"pyside6-uic {UI_FILE_PATH} > {COMPILED_UI_FILE_PATH}")
```

Comment:
Are Python build utilities really so bad that we need this? More proof of Ruby superiority.

Reply:
It's quite unfortunate. The syntax of PyQt6 was this

```py
class MainWindow
    loadUiFile(into MainWindow)
    # keep all of MainWindow's methods
```

Unfortunately, PySide6 syntax (for directly loading a .ui file) is like this

```
MainWindow = loadUi(ui file)
```

There does not seem to be a way around that in PySide6. Someone implemented the PyQt6 syntax in PySide (https://stackoverflow.com/questions/14892713/how-do-you-load-ui-files-onto-python-classes-with-pyside/14894550#14894550) but that's code from a long time ago. Did not get it to work.